src/frontend/src/app/pipes/moment.ts

```diff
@@ -0,0 +1,20 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import * as moment from 'moment';
+
+/*
+ * Time helper using momentjs
+ * Usage:
+ *   timestamp | moment:'DD.MM.YYYY'
+ * Defaults to 'L' - locale ie. '01/24/2017'
+*/
+@Pipe({name: 'moment'})
```

Comment:
pipe name is bit confusing, rename it to more relevant  name i.e. `dateparser` because `moment` defines the library itself imported above.